R/sidebar.R

```diff
@@ -47,15 +48,27 @@ sidebar <- function(
     class <- c("bslib-sidebar-input", class)
   }
 
+  if (is.null(fg) && !is.null(bg)) {
+    fg <- get_color_contrast(bg)
+  }
+  if (is.null(bg) && !is.null(fg)) {
+    bg <- get_color_contrast(fg)
+  }
+
   hide_collapse <- isTRUE(is.na(open))
 
   collapse_tag <- tags$button(
     class = "collapse-toggle",
     type = "button",
     title = "Toggle sidebar",
-    style = css(display = if (hide_collapse) "none"),
+    style = css(
+      display = if (hide_collapse) "none",
+      background_color = bg,
+      color = fg
```

Comment:
Could this just inherit from the parent?

Reply:
I tackled this in e8c991f1ae673e2fff681ddd6d6d3027059275cc. I ended up wiring up a new SASS variable to help theme the default sidebar background color. (See point 9 in the main PR description.)